code/modules/mob/living/carbon/xenomorph/castes/Shrieker.dm

```diff
@@ -0,0 +1,119 @@
+/datum/caste_datum/shrieker
+	caste_type = XENO_CASTE_SHRIEKER
+	caste_desc = "A loud, booming caste that wroughts destruction, or brings support with its shriek."
+	tier = 3
+
+	melee_damage_lower = XENO_DAMAGE_TIER_3
+	melee_damage_upper = XENO_DAMAGE_TIER_3
+	max_health = XENO_HEALTH_TIER_9
+	plasma_gain = XENO_PLASMA_GAIN_TIER_8
+	plasma_max = XENO_PLASMA_TIER_8
+	xeno_explosion_resistance = XENO_EXPLOSIVE_ARMOR_TIER_6
+	armor_deflection = XENO_ARMOR_TIER_2
+	evasion = XENO_EVASION_NONE
+	speed = XENO_SPEED_TIER_7
+
+	evolution_allowed = FALSE
+
+	deevolves_to = list(XENO_CASTE_CARRIER, XENO_CASTE_HIVELORD)
+
+	can_vent_crawl = FALSE
+	can_hold_facehuggers = TRUE
+	can_hold_eggs = CAN_HOLD_TWO_HANDS
```

Comment:
shrieker is big it should be able to hold two eggs :)

Reply:
Pretty sure it does wit this tho